addon/components/bs-accordion.js

```diff
@@ -2,7 +2,7 @@ import Ember from 'ember';
 import layout from '../templates/components/bs-accordion';
 import listenTo from '../utils/listen-to-cp';
 
-const { K: noop } = Ember;
+const {} = Ember;
```

Comment:
Hey cool, I knew I would have to update things after the RFC was accepted, but you were faster! Thanks! :)

Unfortunately the codemod will leave empty destructuring statements like the above, which causes failing JSHint tests. I can fix that manually, but maybe there is a way to fix that also in the codemod?